exporter/logzioexporter/logziospan.go

```diff
@@ -18,20 +17,20 @@ const (
 
 // logzioSpan is same as esSpan with a few different json field names and an addition on type field.
 type logzioSpan struct {
-	TraceID         dbmodel.TraceID     `json:"traceID"`
-	OperationName   string              `json:"operationName,omitempty"`
-	SpanID          dbmodel.SpanID      `json:"spanID"`
-	References      []dbmodel.Reference `json:"references"`
-	Flags           uint32              `json:"flags,omitempty"`
-	StartTime       uint64              `json:"startTime"`
-	StartTimeMillis uint64              `json:"startTimeMillis"`
-	Timestamp       uint64              `json:"@timestamp"`
-	Duration        uint64              `json:"duration"`
-	Tags            []dbmodel.KeyValue  `json:"JaegerTags,omitempty"`
-	Tag             map[string]any      `json:"JaegerTag,omitempty"`
-	Logs            []dbmodel.Log       `json:"logs"`
-	Process         dbmodel.Process     `json:"process,omitempty"`
-	Type            string              `json:"type"`
+	TraceID         TraceID        `json:"traceID"`
+	SpanID          SpanID         `json:"spanID"`
+	OperationName   string         `json:"operationName,omitempty"`
+	References      []reference    `json:"references"`
+	Flags           uint32         `json:"flags,omitempty"`
+	StartTime       uint64         `json:"startTime"`
+	StartTimeMillis uint64         `json:"startTimeMillis"`
+	Timestamp       uint64         `json:"@timestamp"`
+	Duration        uint64         `json:"duration"`
+	Tags            []keyValue     `json:"JaegerTags,omitempty"`
+	Tag             map[string]any `json:"JaegerTag,omitempty"`
+	Logs            []spanLog      `json:"logs"`
+	Process         process        `json:"process,omitempty"`
+	Type            string         `json:"type"`
 }
```

Comment:
also, since you need to reference moved types like TraceID, Process, etc., you can define them in internal/dbmodel package, where they can remain public but not usable outside of this exporter, and the code referring to them will not need to change, just the import.